From d5823c962d7831bbef218c2718955b6da40f798f Mon Sep 17 00:00:00 2001
From: James Page <james.page@chainguard.dev>
Date: Mon, 2 Feb 2026 13:00:59 +0000
Subject: [PATCH] [build] add missing includes (#41351)

Partial cherry-pick of d54219b508423f0a2ff6a0b98c16fb6dafd44b84

(excludes property_list.cc which does not exist in this release)
---
 include/grpc/event_engine/memory_request.h | 2 ++
 src/core/util/glob.cc                      | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/include/grpc/event_engine/memory_request.h b/include/grpc/event_engine/memory_request.h
index 76bcbb2036..ad8cab842e 100644
--- a/include/grpc/event_engine/memory_request.h
+++ b/include/grpc/event_engine/memory_request.h
@@ -17,6 +17,8 @@
 #include <grpc/support/port_platform.h>
 #include <stddef.h>
 
+#include <string>
+
 #include "absl/strings/string_view.h"
 
 namespace grpc_event_engine {
diff --git a/src/core/util/glob.cc b/src/core/util/glob.cc
index 1b1c16e23d..22e54259f0 100644
--- a/src/core/util/glob.cc
+++ b/src/core/util/glob.cc
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <algorithm>
+
 #include "absl/strings/string_view.h"
 
 namespace grpc_core {
-- 
2.51.0

